demo/demo.sh

```diff
@@ -25,8 +25,8 @@ run () {
         echo "$title '$t'; $(cprofile ${BASE_PATH}/$@) 2>&1 | tee ${BASE_PATH}/$t.log; sleep infinity"
 }
 
-run_router="$(run router soma-subscriptions-producer)"
```

Comment:
In the interests of getting the massive rename commit in before things pile up behind it, can the ‘do this right’ change be deferred to a follow-up commit here?